doc/maintainer-guide.md

```diff
@@ -0,0 +1,37 @@
+# Reviewing PRs
+
+Approving a PR means that the reviewer has deemed the PR ready to be merged.
+
+There are two steps to reviewing PRs.
+
+- Review the feature:
+
+  - Consider if the feature aligns with the [InfiniTime vision](InfiniTimeVision.md)
+  - Discuss alternative ideas or implementations
+
+- Review the code:
+
+  - Check the quality of the code and make sure it conforms to the [coding conventions](coding-convention.md)
+  - Consider the maintainability of the code
+  - Test the code with at least InfiniSim or a PineTime
+
+# Merging PRs
+
+Two approvals from core developers is usually required to merge a PR.
+Exceptions include urgent fixes
+and small maintenance PRs by core developers that don't affect the apparent behaviour of the watch in any way.
+Then only a single approval is needed.
+
+Commits that fix code formatting from previous commits must be squashed before merging a PR,
+or the PR must be squash merged.
+Otherwise the PR shall be squash or rebase merged.
+
+All but the simulator check must be passed before merging a PR.
```

Comment:
@JF002 There are branch protection rules which make it so checks must be passed before merging. If these are enabled for build-firmware, clang-format and clang-tidy, this line can be removed.

Reply:
Do you mean this for the **develop** branch? 
![image](https://user-images.githubusercontent.com/2261652/212159133-a8cf0fae-eb35-4369-b653-29256202ea9e.png)

Enabling as many auto checks as possible is probably a good idea to make our lifes easier, but I also think it's a good idea to mention those checks in the maintainer's guide, as a documentation of the process.

Reply:
Okay, I'll keep the note here then. I'll also keep the wording, even though it doesn't exactly match the branch protection rules.